### PR TITLE
Fix/em yo

### PR DESCRIPTION
**Pull request by Scrum**
## Description

- Fixes *"Yo"* not outputted correctly, issue #1358

## Contributor

- [x] Test(s) exist to ensure functionality and minimize regression (if no tests added, list tests covering this PR)

## Committer

In most cases, this should be a different person than the contributor.

- [ ] Draft GitHub release notes have been updated.
- [ ] CI is green (no forced merge required).
- [ ] Merge PR

### CODE REVIEW

File: test/specs/gfm/commonmark.0.29.json [hunk 1]
```diff
@@ -2850,7 +2850,7 @@
   },
   {
     "markdown": "a*\"foo\"*\n",
-    "html": "<p>a*&quot;foo&quot;*</p>\n",
+    "html": "<p>a<em>&quot;foo&quot;</em></p>\n",
```
Comment (UziTech) [inline test/specs/gfm/commonmark.0.29.json:2853]: according to the [commonmark spec](https://spec.commonmark.org/0.29/#example-352) that should not change.

Comment (UziTech) [reply test/specs/gfm/commonmark.0.29.json:2853]: These common mark tests are derived from the spec so they should not be changed unless a new version of the spec comes out.

If a spec is marked with `shouldFail: true` that means it is failing and needs to be fixed in marked.

Comment (Scrum) [reply test/specs/gfm/commonmark.0.29.json]: marked as shouldFail

**Comment by UziTech** [comment]
Seems like this is moving backwards in spec compliance. Could you find a way to fix `"yo"` while not failing spec tests that were passing?

**Comment by Scrum** [comment]
I don't really understand

1. https://spec.commonmark.org/dingus/?text=*%22Yo%22* here it is clearly visible that he is working out correctly, I don't know where the error is

2. I don't really understand the problem that needs to be solved so that the same value is handled differently

**Comment by UziTech** [comment]
The problem is that <https://spec.commonmark.org/dingus/?text=a*%22Yo%22*> is not working correctly with this PR but it is working without this PR.

**Comment by Scrum** [comment]
> The problem is that https://spec.commonmark.org/dingus/?text=a*%22Yo%22* is not working correctly with this PR but it is working without this PR.

Screenshot 2020-04-29 at 09 08 58

> Can you find a way to make *"Yo"* = <em>"Yo"</em> work as well as a*"Yo"* = a*"Yo"*?

To do this, just return the rule to the state of the master, but then this PR has no value

**Comment by UziTech** [comment]
> To do this, just return the rule to the state of the master, but then this PR has no value

`*"Yo"* = <em>"Yo"</em>` isn't working in master

**Comment by UziTech** [comment]
`marked('*"Yo"*')` is supposed to output `<em>"Yo"</em>` which works with this PR but in master it outputs `*"Yo"*`

`marked('a*"Yo"*')` is supposed to output `a*"Yo"*` which works in master but not with this PR.

Can you update this PR to make both work with this PR?

**Comment by Scrum** [comment]
no since it is one and the same `marked('*"Yo"*')` `marked('a*"Yo"*')` you see the difference here if you miss the symbol `a` ?

**Comment by UziTech** [comment]
It is definitely possible. You will have to change more than just the `em` regex to make it work.

I don't think this PR should be merged if it could break current spec compliant markdown.

**Pull request closed without merging**